src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Rand.txt

```diff
@@ -0,0 +1,28 @@
+// Test Rand() - beware, rand is volatile. 
+
+>>RandBetween(1, 1)
+1
+
+>>IsError(RandBetween(5, 1))
+true
+
+>>RandBetween(Blank(),1)
+0
```

Comment:
The result here can be either 0 or 1 - `RandBetween(a,b)` should return a random number between a and b inclusive.

Reply:
just moving preexisting tests, fix in separate PR.